.gitignore

```diff
@@ -79,3 +79,4 @@ yarn-error.log*
 .gradle
 gradle
 Dockerrun.aws.json
+prisonstaffhub-specs/out
```

Comment:
Found it strange that I needed to add this.  Let me know if there's something I'm missing